src/contracts/core/PaymentCoordinatorStorage.sol

```diff
@@ -30,6 +30,10 @@ abstract contract PaymentCoordinatorStorage is IPaymentCoordinator {
     uint32 public immutable MAX_FUTURE_LENGTH;
     /// @notice absolute min timestamp (seconds) that a payment can start at
     uint32 public immutable GENESIS_PAYMENT_TIMESTAMP;
+    /// @notice The cadence at which a snapshot is taken offchain for calculating payment distributions
+    uint32 public constant SNAPSHOT_CADENCE = 1 days;   
+    /// @notice The maximum payment token amount for a single range payment, constrained by off-chain calculation
+    uint256 public constant MAX_PAYMENT_AMOUNT = 1e38 - 1;
```

Comment:
For clarity and consistency with other constants, should be added to the interface as well